Tests/APITesters/SwiftAPITester/SwiftAPITester/PresentedOfferingContextAPI.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  PresentedOfferingContextAPI.swift
+//  SwiftAPITester
+//
+//  Created by Josh Holtz on 2/14/24.
+//
+
+import Foundation
+import RevenueCat
+import StoreKit
+
+private var context: PresentedOfferingContext!
+func checkPresentedOfferingContextAPI() {
+    let oID: String = context.offeringIdentifier
+
+    print(context!, oID)
```

Comment:
We've been simplifying these:
```suggestion
func checkPresentedOfferingContextAPI(context: PresentedOfferingContext! = nil) {
    let _: String = context.offeringIdentifier
}
```